transcribe_demo.py

```diff
@@ -106,22 +107,16 @@ def record_callback(_, audio:sr.AudioData) -> None:
                     phrase_complete = True
```

Comment:
Remove `last_sample = bytes()` on the above line, this variable is no longer used.

Reply:
done